internal/command/providers.go

```diff
@@ -49,7 +49,7 @@ func (c *ProvidersCommand) Run(args []string) int {
 
 	var diags tfdiags.Diagnostics
 
-	empty, err := configs.IsEmptyDir(configPath)
+	empty, err := configs.IsEmptyDir(configPath, "")
```

Comment:
We do also load providers from the testing files - should we also include the testing directory here as well?

Reply:
Oh Yeah. Sounds goog 👍